src/App.vue

```diff
@@ -1,24 +1,66 @@
 <template>
-  <v-app>
-    <Navbar />
+  <v-app v-if="isLoggedIn">
+    <Navbar/>
       <v-container>
         <div class="content">
           <router-view />
         </div>
       </v-container>
   </v-app>
+  <v-app v-else>
+    <router-view v-if="isLoginTryCommited"/>
+  </v-app>
 </template>
 
 <script lang="ts">
+import { getModule } from 'vuex-module-decorators';
 import { Component, Vue } from 'vue-property-decorator';
-import Navbar from './components/AppComponents/Navbar.vue';
+
+import StoreModule from '@/store/storeModule';
+import Navbar from '@/components/AppComponents/Navbar.vue';
+import User from '@/models/User';
+import { getLoggedInUser } from '@/db-service/Users/queries';
 
 @Component({
   components: {
     Navbar,
   },
 })
-export default class App extends Vue {}
+export default class App extends Vue {
+  private storeModule = getModule(StoreModule, this.$store);
+
+  isLoginTryCommited = false;
+
+  public mounted() {
+    const userName: string | null = localStorage.getItem('userName');
+    const password: string | null = localStorage.getItem('password');
+
+    if (userName !== null && password !== null) {
+      this.$apollo.query({
+        query: getLoggedInUser,
+        variables: {
+          userName,
+          password,
+        },
+      }).then((data) => {
+        if (data.data.loggedInUser !== undefined) {
+          const loggedInUsers: User[] = data.data.loggedInUser.nodes;
+          if (loggedInUsers.length !== 0) {
+            this.storeModule.setUserId(loggedInUsers[0].id.toString());
+            this.$router.back();
+          }
+        }
+      }).finally(() => { this.isLoginTryCommited = true; });
+    } else {
+      this.isLoginTryCommited = true;
+    }
+  }
+
+  get isLoggedIn() {
+    const isUserLogged: boolean = this.storeModule.userId !== '';
```

Comment:
Maybe you can use a getter for the userId in the StoreModule because there is a need to keep it in readonly property, like in ...mapState(['']) that is usually used in computed

Reply:
Agree